packages/api/src/platforms/vtex/utils/shouldUpdateShippingData.ts

```diff
@@ -0,0 +1,50 @@
+import { IStoreSession } from '../../../__generated__/schema'
+import { OrderForm } from '../clients/commerce/types/OrderForm'
+
+export const shouldUpdateShippingData = (
```

Comment:
Overall, I'd improve this function by moving each condition (hasDifferentPostalCode, hasDifferentGeoCoordinates and hasDifferentDeliveryMode) to separate functions. They will not be used separately, but the behavior will be far easier to understand.